ui-ngx/src/assets/locale/locale.constant-en_US.json

```
@@ -7073,7 +6723,8 @@
         "runtime-failed": "<b>Failed:</b> {{message}}",
         "auto-commit-settings-read-only-hint": "Auto-commit feature doesn't work with enabled read-only option in Repository settings.",
         "rollback-on-error": "Rollback on error",
-        "rollback-on-error-hint": "If you have a large amount of entities to restore, consider disabling this option to increase performance.\n Note, if an error occurs over the course of version loading, already persisted entities (with relations, attributes, etc.) will stay as is"
+        "rollback-on-error-hint": "If you have a large amount of entities to restore, consider disabling this option to increase performance.\n Note, if an error occurs over the course of version loading, already persisted entities (with relations, attributes, etc.) will stay as is",
+        "version-name-max-length": "Version name should be less than 256"
```

```suggestion
        "version-name-max": "Version name should be less than 256"
```
and move to another version-name translation key